cime_config/machines/config_machines.xml

```diff
@@ -4215,7 +4215,7 @@ commented out until "*** No rule to make target '.../libadios2pio-nm-lib.a'" iss
     <DIN_LOC_ROOT_CLMFORC>/usr/projects/e3sm/inputdata/atm/datm7</DIN_LOC_ROOT_CLMFORC>
     <DOUT_S_ROOT>/lustre/scratch5/$ENV{USER}/E3SM/archive/$CASE</DOUT_S_ROOT>
     <BASELINE_ROOT>/lustre/scratch5/$ENV{USER}/E3SM/input_data/ccsm_baselines/$COMPILER</BASELINE_ROOT>
-    <CCSM_CPRNC>/usr/projects/climate/SHARED_CLIMATE/software/badger/cprnc</CCSM_CPRNC>
+    <CCSM_CPRNC>/usr/projects/e3sm/software/chicoma-cpu/cprnc</CCSM_CPRNC>
```

Comment:
Oh, I bet I know what happened here.  I deleted this thinking that it was old and no longer used.  In my defense, it has `badger` in the path...

Reply:
And it no longer exists! Not just the machine but that file. But I think if that line is missing it forces each test to try to build it

Reply:
> And it no longer exists! 

That's what I was saying.  I think I deleted it trying to free up space in `/usr/projects/climate` because I couldn't imagine we were still using software built for Badger.